src/services/FontAwesomeService/index.js

```diff
@@ -117,7 +117,7 @@ class FontAwesomeService {
 	];
 	static #brandIconsBlackList = ['faYandexInternational', 'faYandex', 'faVk', 'faSquareOdnoklassniki', 'faOdnoklassniki'];
 
-	static #getIconVisibleName(name) {
+	static getIconVisibleName(name) {
```

Comment:
This and other services are static classes. I just liked this concept after writing a project on C# a while ago. But you are right, that is not the javascript approach. Made most of the services use ES6 modules, named exports, and renamed functions.
The only service I had to keep as a static class is `FaviconGrabberService` because using named exports breaks mocking in unit tests for `tryAllPossibleCases`.